### PR TITLE
Fix: #15 Second example disconnects immediately

### PR DESCRIPTION
**Pull request by mpb**
Fix: #15 Second example disconnects immediately which means nothing is delivered

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -110,7 +111,6 @@
 
     // subscribe to Frame-Timing and User-Timing events
     observer.observe({eventTypes: ['render', 'composite', 'mark', 'measure']});
-    observer.disconnect();
 
     // processing logic called by application code
     function processObservedPerfEvents() {
```
Comment (igrigorik) [inline index.html]: The example never calls `processObservedPerfEvents()`.. The idea was to provide a "defer processing" example, but FWIW, perhaps that's unnecessary as a first intro to the API. My proposal would be to merge contents of that function into the PO callback?

Comment (mpb) [inline index.html]: PTAL

File: index.html [hunk 2]
```diff
@@ -103,23 +103,35 @@
     <p>Alternatively, instead of processing metrics at a predefined time, or having to periodically poll the timeline for new metrics, the developer may also observe the <a>Performance Timeline</a> and be notified of new performance metrics via a <a>Performance Observer</a>:</p>
 
     <pre class="highlight example">
-    var observedEvents = [];
+&lt;!doctype html&gt;
+&lt;html&gt;
+  &lt;head&gt;
+  &lt;/head&gt;
+  &lt;body onload="init()"&gt;
```
Comment (igrigorik) [inline index.html]: onload is unnecessary, otherwise lgtm!

Comment (mpb) [inline index.html]: Better?

**Comment by igrigorik** [comment]
lgtm, thanks!

**Pull request merged**